### PR TITLE
Add "Squiggle" plugin

### PR DESCRIPTION
**Pull request by jqhoogland**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin:  https://github.com/jqhoogland/obsidian-squiggle

## Release Checklist
- [ ] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [x]  Linux
  - [ ]  ~~Android _(if applicable)_~~
  - [ ]  ~~iOS _(if applicable)_~~
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by liamcain** [comment]
- [this.stdoutElem.innerHTML = this.stdoutText](https://github.com/jqhoogland/obsidian-squiggle/blob/e66ebf5ed44a70fc3d4711a1269866ce940caaa2/src/Outputter.ts#L48) [innerHTML is a potential security risk](https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md#avoid-innerhtml-outerhtml-and-insertadjacenthtml). If you need to use it, you should use it with [`sanitizeHTMLToDom`](https://github.com/obsidianmd/obsidian-api/blob/6b2138aa28fb6d6f58f78e6fabec02186fdc0f82/obsidian.d.ts#L2977).
- [console.log(this.app)](https://github.com/jqhoogland/obsidian-squiggle/blob/e66ebf5ed44a70fc3d4711a1269866ce940caaa2/src/main.ts#L19) Remove this

**Comment by jqhoogland** [comment]
- Removed the `innerHTML` call (wasn't even necessary)
- Removed the `console.log`

File: community-plugins.json [hunk 1]
```diff
@@ -4838,5 +4838,12 @@
         "author": "Ellpeck",
         "description": "Multi-purpose time trackers for your notes!",
         "repo": "Ellpeck/ObsidianSimpleTimeTracker"
+    },
+    {
+        "id": "squiggle",
+        "name": "Squiggle",
+        "author": "Jesse Hoogland",
+        "description": "Display a list of recently opened files",
```
Comment (liamcain) [inline community-plugins.json:4846]: I think this is the wrong description for this plugin? Do you mind changing this to be descriptive of what your plugin does, then I can merge it? Thanks!

Comment (jqhoogland) [reply community-plugins.json]: Yep! It should be good now. Thanks for your patience!

**Pull request merged**